frontend/app/routes/$lang+/_public+/apply+/$id+/communication-preference.tsx

```diff
@@ -63,7 +63,16 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const csrfToken = String(session.get('csrfToken'));
   const meta = { title: t('gcweb:meta.title.template', { title: t('apply:communication-preference.page-title') }) };
 
-  return json({ communicationMethodEmail, id: state.id, csrfToken, meta, preferredCommunicationMethods, preferredLanguages, defaultState: state.communicationPreferences, editMode: state.editMode });
+  return json({
+    communicationMethodEmail,
+    id: state.id,
+    csrfToken,
+    meta,
+    preferredCommunicationMethods,
+    preferredLanguages,
+    defaultState: { ...state.communicationPreferences, email: state.personalInformation?.email ?? state.communicationPreferences?.email, confirmEmail: state.personalInformation?.confirmEmail ?? state.communicationPreferences?.confirmEmail },
```

Comment:
Fallback to personalInformation email if communicationPreferences email is not defined.

Nitpick: put this line on multple lines for readability.

```suggestion
    defaultState: {
      ...state.personalInformation,
      email: state.communicationPreferences?.email ?? state.personalInformation?.email,
      confirmEmail: state.communicationPreferences?.confirmEmail ?? state.personalInformation?.confirmEmail,
    },
```

Reply:
Fallback to personalInformation email if communicationPreferences email is not defined.

``` jsx
email: state.communicationPreferences?.email ?? state.personalInformation?.email,
confirmEmail: state.communicationPreferences?.confirmEmail ?? state.personalInformation?.confirmEmail,
```

Reply:
@Fbasham not fixed yet.